### PR TITLE
BTAT-2015 - Implement Banner feedback  and set Beta Banner

### PR DESCRIPTION
**Pull request by HarryGull**

### CODE REVIEW

**Comment by HarryGull** [comment]
To test locally start service: CONTACT_FRONTEND, DESKPRO_STUB and HMRCDESKPRO

File: app/views/feedback/feedback_thankyou.scala.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever
+@import config.FrontendAppConfig
+
+@(partialUrl: String, redirectUrl: String)(implicit request: Request[_], cachedStaticHtmlPartialRetriever: CachedStaticHtmlPartialRetriever,
+    messages: Messages, appConfig: FrontendAppConfig)
+
+@views.html.main_template(title = messages("openPayments.title"), bodyClasses = None, appConfig = appConfig) {
```
Comment (miranda-hawkes) [inline app/views/feedback/feedback_thankyou.scala.html]: Please update to use correct page title

File: app/views/feedback/feedback.scala.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+@import uk.gov.hmrc.play.partials.FormPartialRetriever
+@import uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever
+@import config.FrontendAppConfig
+
+@(partialUrl: String, formBody: Option[Html])(implicit request: Request[_], formPartialRetriever: FormPartialRetriever,
+partialRetriever: CachedStaticHtmlPartialRetriever, messages: Messages, appConfig: FrontendAppConfig)
+
+
+@views.html.main_template(title = messages("openPayments.title"), bodyClasses = None, appConfig = appConfig) {
```
Comment (miranda-hawkes) [inline app/views/feedback/feedback.scala.html]: Please update to use correct page title

Comment (miranda-hawkes) [inline app/views/feedback/feedback.scala.html]: Same comment here as in view-vat-returns-frontend regarding the page width

File: app/views/govuk_wrapper.scala.html [hunk 3]
```diff
@@ -69,11 +69,10 @@
...
 
 @phaseBanner = {
-        <!--TODO:Update href to feedback controller when in beta-->
-    <div class="alpha-banner">
+    <div class="beta-banner">
         <p>
-            <strong id="phase" class="phase-tag">ALPHA</strong>
-            <span>This is a new service – your <a id="alpha-banner-feedback" href="www.google.com">feedback</a> will help us to improve it.</span>
+            <strong id="phase" class="phase-tag">@Messages("banner.phaseName")</strong>
+            <span>@Messages("banner.newService") <a id="alpha-banner-feedback" href="@controllers.feedback.routes.FeedbackController.show()">@Messages("banner.feedback")</a> @Messages("banner.feedbackImprove")</span>
```
Comment (miranda-hawkes) [inline app/views/govuk_wrapper.scala.html]: Can you update the id to be `beta-banner-feedback` please

**Pull request merged**